src/container/TodoBoardContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { connect } from "react-redux";
+import TodoBoard from "../components/Todo/TodoBoard";
+import {
+  addTodoItem,
+  setTodoItemStatus,
+  setStrikeThrough,
+} from "../features/todos/todosSlice";
+import { filterTodoItems } from "../features/filters/filtersSlice";
+import { selectTodoItemsByStatus } from "../selector";
+import { State, Status } from "../App";
+
+const mapStateToProps = (state: State) => ({ state: state });
+
+function TodoBoardContainer({
+  state,
+  dispatch,
+}: {
+  state: State;
+  dispatch: any;
```

Comment:
should not use `any` if possible

https://dev.to/codiga/why-you-should-not-use-the-type-any-in-typescript-1li7